### DISCUSSION TITLE
20231026 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by josecelano** [nautilus-cyberneering/.github#63] [Meetings]
Date: 2023-10-26

Time: 8:30 UTC (10:30 in Vaduz)

Location:
<https://meeting.hodl10.io/ConsistentClassroomsMergeSideways>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- CGarnham(@da2ce7)  - 
- JCelano(@josecelano) - 
- CBosse(@cgbosse) - 
- MMelchor(@mario-nt) - 
- GByrne(@grmbyrn) - 
- MVanDijke(@WarmBeer) - 

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by josecelano** [comment]
### Torrust Tracker

- Review PR: https://github.com/torrust/torrust-tracker/pull/473
- Discussion Web Torrent support: https://github.com/torrust/torrust-tracker/discussions/138

### Torrust Index

- Discussion Don’t DELETE ON CASCADE: https://github.com/torrust/torrust-index/discussions/356

### Torrust Website

- New blog post: "Dockerizing Rust Applications: Best Practices" **WIP**

### Nautilus

Twitter profile: https://twitter.com/nautilus_cyber

### Rust

New Meetup in Las Palmas de GC: https://www.meetup.com/las-palmas-rust-meetup-group/

**Comment by grmbyrn** [comment]
## Torrust Website
* Created a route to sample profile page for authors of blog posts
* Created new issue on adding contact form to homepage https://github.com/torrust/torrust-website/issues/92
* Continued with feature to copy code blocks when clicking on button. Reached out for assistance from SK community for help with it

## Secure Git Guide
* Moved file to root and deleted unnecessary code
* Added blog post on how to use Git and GitHub from root into the articles route

## Learning
* Continued learning Python - on learning path through purchase from Codecademy
* Revising Svelte and SvelteKit to drill what I have learned

**Comment by mario-nt** [comment]
### GUI
**Troubleshooting and fixing CI errors and E2E test failing on the the following issues:**
- Worked on https://github.com/torrust/torrust-index-gui/issues/229
- Worked on https://github.com/torrust/torrust-index-gui/issues/232
- Worked on https://github.com/torrust/torrust-index-gui/issues/310
- Finished working on https://github.com/torrust/torrust-index-gui/issues/278
### INDEX
- Worked on https://github.com/torrust/torrust-index/issues/39
- Finished working on https://github.com/torrust/torrust-index/issues/303
### Other
-Troubleshooting CI errors, found out  that the Docker image for the Index was not built, so the database used in the E2E tests didn't exist
-Fixed errors while running the Torrust index and E2E test for the Torrust gui, the env variables to allow the CORS policy changed from TORRUST_IDX_BACK_CORS_PERMISSIVE to TORRUST_INDEX_BACK_CORS_PERMISSIVE.
-Fixed Torrust index not compiling dependencies pacakages by updating Rust to latest version

**Comment by mickvandijke** [comment]
### Torrust Tracker

- Finished working on https://github.com/torrust/torrust-tracker/pull/497
- Worked on https://github.com/torrust/torrust-tracker/issues/495
